pbench/server/hierarchy.py

```diff
@@ -59,14 +84,22 @@ def __init__(self, name, path, config):
         self.bad_controllers = list()
         self.tarballs = list()
         self.things_to_check = {
-            "unexpected_symlinks": DictOfList(),
-            "unexpected_objects": DictOfList(),
-            "expected_objects": DictOfList(),
-            "unexpected_dirs": DictOfList(),
-            "non_prefixes": DictOfList(),
-            "wrong_prefixes": DictOfList(),
-            "non_prefix_dirs": DictOfList(),
-            "status_indicators": DictOfList(),
+            "unexpected_dirs": DictOfList(
+                "Unexpected state directories found in this controller directory:"
+            ),
+            "status_indicators": DictOfList(""),
```

Comment:
Split status indicators into separate DictOfList()s so that the loop on ll.152-153 will work for them as well and we won't need to change gold files.